packages/nextjs/components/Header.tsx

```diff
@@ -90,11 +91,11 @@ export const Header = () => {
         </div>
         <Link href="/" passHref className="hidden lg:flex items-center gap-2 ml-4 mr-6 shrink-0">
           <div className="flex relative w-10 h-10">
-            <Image alt="SE2 logo" className="cursor-pointer" fill src="/logo.svg" />
+            <Image alt="SE2 logo" className="cursor-pointer" fill src={projectConfig.iconPathHeader} />
```

Comment:
I don't see a reason to change logo path. Why just not remove default `logo.svg` and add a new one? It also helps to not forget about removing old one